Add Secrets to game-settings.md (#845)
Roblox/creator-docs: content/en-us/studio/game-settings.md

    <td>**Allow HTTP Requests**</td>
    <td>Allows experiences' servers to issue requests to remote servers via `Class.HttpService`.</td>
  </tr>
  <tr>
    <td>**Secrets**</td>

>     <td>Allows for the creation and configuration of [Secrets](../reference/engine/datatypes/Secret.yaml) for Roblox Studio sessions using `Class.HttpService`. Requires allowing HTTP requests.</td>

Reviewer: ```suggestion
    <td>Allows for the creation and configuration of [Secrets](../reference/engine/datatypes/Secret.yaml) for Studio sessions using `Class.HttpService`. Visible only if HTTP requests are allowed.</td>
```
